Fix numbering in the README file (#11)
ballerina-guides/integration-tutorials: send-emails-from-a-service/README.md

The flow is as follows.

> 1. Receive a request with a JSON payload similar to the following form.

Reviewer: ```suggestion
1. Receive a request with a JSON payload in the following form.
```

Since it is the structure we are talking about, doesn't need similar IMO.

}
```

> 3. Call the payment backend service to make the payment and retrieve the reservation response which will have a payload similar to the following. If the payment is successful, send an email to the user with the appointment details.

Reviewer: Why do we need this change?

Author: Thought it would be better to use `following` since that's what we've used everywhere else

Reviewer: Would have been OK if we didn't have the next sentence. But since we have it I prefer the former version.
